src/components/calculator/Calculator.js

```diff
@@ -1,163 +1,158 @@
+        value="x"
+        type="button"
+        className="calculator__key calculator__key--operator calculator__padding"
+        onClick={handleClick}
+      />
 
-        <input
-          value="4"
-          type="button"
-          className="calculator__key calculator__padding"
-          onClick={this.handleClick}
-        />
+      <input
+        value="4"
+        type="button"
+        className="calculator__key calculator__padding"
+        onClick={handleClick}
+      />
 
-        <input
-          value="5"
-          type="button"
-          className="calculator__key calculator__padding"
-          onClick={this.handleClick}
-        />
+      <input
+        value="5"
+        type="button"
+        className="calculator__key calculator__padding"
+        onClick={handleClick}
+      />
 
-        <input
-          value="6"
-          type="button"
-          className="calculator__key calculator__padding"
-          onClick={this.handleClick}
-        />
+      <input
+        value="6"
+        type="button"
+        className="calculator__key calculator__padding"
+        onClick={handleClick}
+      />
 
-        <input
-          value="-"
-          type="button"
-          className="calculator__key calculator__key--operator calculator__padding"
-          onClick={this.handleClick}
-        />
+      <input
+        value="-"
+        type="button"
+        className="calculator__key calculator__key--operator calculator__padding"
+        onClick={handleClick}
+      />
 
-        <input
-          value="1"
-          type="button"
-          className="calculator__key calculator__padding"
-          onClick={this.handleClick}
-        />
+      <input
+        value="1"
+        type="button"
+        className="calculator__key calculator__padding"
+        onClick={handleClick}
+      />
 
-        <input
-          value="2"
-          type="button"
-          className="calculator__key calculator__padding"
-          onClick={this.handleClick}
-        />
+      <input
+        value="2"
+        type="button"
+        className="calculator__key calculator__padding"
+        onClick={handleClick}
+      />
 
-        <input
-          type="button"
-          className="calculator__key calculator__padding"
-          value="3"
-          onClick={this.handleClick}
-        />
+      <input
+        type="button"
+        className="calculator__key calculator__padding"
+        value="3"
+        onClick={handleClick}
+      />
 
-        <input
-          className="calculator__key calculator__key--operator calculator__padding"
-          value="+"
-          type="button"
-          onClick={this.handleClick}
-        />
-        <input
-          value="0"
-          type="button"
-          className="calculator__key calculator__key--zero calculator__padding"
-          onClick={this.handleClick}
-        />
+      <input
+        className="calculator__key calculator__key--operator calculator__padding"
```

Comment:
* I cant perform division operation kindly fix it 🙏 (ie 30 divided by 10)

Reply:
@ Davidosky007 i have made the required changes